src/js/modules/core/components/MastheadBar.js

```diff
@@ -67,36 +72,62 @@ const navButtonStyles = {
     fontSize: '12px',
     fontWeight: 'bold',
     textTransform: 'uppercase',
+  },
+  barsIcon: {
+    marginRight: '6px',
+  },
+  searchIcon: {
+    marginRight: '3px',
   }
 };
 
-const NavButton = ({ classes, label }) => {
+const NavButton = ({ classes, label, onClick }) => {
+  const createIcon = () => {
```

Comment:
Why not just render FaBars as a child component of NavButton?

\<NavButton>
\<Fabars>
\</NavButton>